_data/apps.yml

```diff
@@ -1298,3 +1298,15 @@
     - "Unity"
     - "Scratch"
   icon: "gausssense.png"
+-
+  name: "Jasper"
+  description: "The GitHub Issue Reader"
```

Comment:
Since this isn't affiliated with GitHub the description here and on the site should be rephrased to something like "The Issue Reader for GitHub". Other than that the pull request looks good.